src/content/docs/accounts/accounts-billing/new-relic-one-pricing-billing/new-relic-one-pricing-billing.mdx

```diff
@@ -45,18 +47,19 @@ In addition to the primary billing factors mentioned above, there are several op
 * Exceeding the [default data retention time](/docs/data-apis/manage-data/manage-data-retention/#adjust-retention)
 * [Vulnerability management](/docs/vulnerability-management/overview): this is included with [Data Plus](/docs/accounts/accounts-billing/new-relic-one-pricing-billing/data-ingest-billing#data-prices), but it's an add-on without Data Plus.
 
-## What do you get for free? [#free]
+## Free tier details [#free]
 
-If you're below the [free limits](#how-pricing-works), you can use New Relic free, forever. Here's a summary of what you get on the free tier:
+You can use New Relic free, forever, if you're okay with what you get on the free tier. Here's what you get for free:
```

Comment:
I'd delete "if you're okay with". it's a little too... something.

I think it's fine as "You can use New Relic free, forever. Here's what you get:"